packages/amplify-category-hosting/lib/S3AndCloudFront/index.js

```diff
@@ -53,22 +52,8 @@ async function enable(context) {
 }
 
 async function checkCDN(context) {
-  const answer = await prompter.pick('Select the environment setup:', Environments, { initial: byValue(DEV) });
-  if (answer === DEV) {
-    removeCDN(context);
-  } else {
-    makeBucketPrivate(context);
-  }
-}
-
-function removeCDN(context) {
-  delete context.exeInfo.template.Resources.OriginAccessIdentity;
-  delete context.exeInfo.template.Resources.CloudFrontDistribution;
-  delete context.exeInfo.template.Resources.PrivateBucketPolicy;
-  delete context.exeInfo.template.Outputs.CloudFrontDistributionID;
-  delete context.exeInfo.template.Outputs.CloudFrontDomainName;
-  delete context.exeInfo.template.Outputs.CloudFrontSecureURL;
-  delete context.exeInfo.template.Outputs.CloudFrontOriginAccessIdentity;
+  await prompter.pick('Select the environment setup:', Environments, { initial: byValue(PROD) });
```

Comment:
can this prompt be removed completely? We aren't using the response value anymore

Reply:
Given the changes here, it looks like `amplify-console-integration-tests` may need updating: https://github.com/aws-amplify/amplify-cli/blob/dev/packages/amplify-console-integration-tests/src/consoleHosting/consoleHosting.ts#L34

Reply:
I think this is a different plugin, namely amplify-console-hosting for this plugin

- Hosting with Amplify Console (Managed hosting with custom domains, Continuous deployment)